index.js

```diff
@@ -1,82 +1,135 @@
 'use strict';
 
-function constant(value) {
-    return function() {
-        return value;
+var GLOBAL_ATTRIBUTE_PREFIX = '$';
+
+module.exports = create;
+module.exports.is = is;
+
+function create(parameters) {
+    var property = parameters.property !== undefined ? parameters.property : '$zoom';
+
+    var inner, outer;
+    if (!is(parameters)) {
+        inner = function() { return parameters; };
+        outer = function() { return inner; };
+        inner.isConstant = outer.isConstant = true;
+
+    } else if (property[0] === GLOBAL_ATTRIBUTE_PREFIX) {
+        outer = function(attributes) {
+            var value = evaluate(parameters, attributes[property]);
+            inner = function() { return value; };
+            inner.isConstant = true;
+            return inner;
+        };
+
```

Comment:
Ok. I was just going with `undefined == false` but its probably good to be explicit here.